src/storybook/config.js

```diff
@@ -2,6 +2,10 @@
 import requireContext from "require-context.macro";
 import { configure } from "@storybook/react";
 
+window.__SALEOR_CONFIG__ = {
```

Comment:
I think we should fix case that `__SALEOR_CONFIG__` can be undefined in `webpack.d.ts` and add fix inside this function: https://github.com/saleor/saleor-dashboard/blob/3247131570213dc246385b382ee337f20bb27a5c/src/config.ts#L7

Reply:
well.... storybook should not even touch API endpoints but we will manage that later

Reply:
if we provide default values for storybook we should just have:
```js
window.__SALEOR_CONFIG__ = {
	APP_MOUNT_URI: "/"
}
```

Reply:
and if we want to make sure to fallback when there is no `__SALEOR_CONFIG__` present we should fix function in `config.ts`